tests/fe/inf_fe_radial_test.C

```diff
@@ -410,15 +1000,15 @@ public:
         {12,9,Point(0.0184475,0.0688471,-0.254748)},
         {13,6,Point(-0.230424,0.230424,1.15264)},
         {14,20,Point(-0.000965042,0.00360159,0.000183379)},
-        {15,15,Point(-0.0423204,0.0113397,0.12514)},
+        {15,15,Point(-0.0423204,0.0113397,0.12514)}
       };
 
     tabulated_grads[std::make_pair(FOURTH, JACOBI_30_00)] =
       {
         {16,3,Point(-0.0476508,0.012768,0.771323)},
         {17,10,Point(0.333487,-0.333487,1.01421)},
         {18,14,Point(0,0,0)},
-        {19,27,Point(-0.00698362,0.00187126,0.000667664)},
+        {19,27,Point(-0.00698362,0.00187126,-0.00228396)},
```

Comment:
The other diffs in this test look like whitespace changes or replacing small numbers with 0, but this line appears to be a real difference? I'm curious how it passed before but also passes now, with a different gold value?

Reply:
this is wrong: I had changed these values but found an error and changed them back; but forgot this one.
Interesting it runs through: the difference between the correct (deleted) and the new value is `0.002951624` in z-coordinate, the `norm_sq()` of the difference thus `8.712084e-06` which is below the threshold of `1e-4`.

Reply:
Ah, yeah I forgot we used a relatively loose tolerance on that test.